src/main/java/com/Nearsoft/referrals/service/impl/RecruitersUpdaterService.java

```diff
@@ -0,0 +1,66 @@
+package com.Nearsoft.referrals.service.impl;
+
+import com.Nearsoft.referrals.model.Recruiter;
+import com.Nearsoft.referrals.repository.RecruiterRepository;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+
+
+@Service
+public class RecruitersUpdaterService {
+    private RecruiterRepository recruiterRepository;
+
+    public RecruitersUpdaterService(RecruiterRepository recruiterRepository) {
+        this.recruiterRepository = recruiterRepository;
+    }
+
+    @PostConstruct
+    public void updateRecruiters() {
```

Comment:
If you wish to load the recruiters at startup time, I recommend to define the data in a yml file, do not put the data hard coded.